bin/reth/src/commands/stage/run.rs

```diff
@@ -118,10 +118,10 @@ impl Command {
                     let mut config = config;
                     config.peers.trusted_nodes_only = self.network.trusted_only;
                     if !self.network.trusted_peers.is_empty() {
```

Comment:
this if can be removed

Reply:
I didn't got this.

Reply:
`if !self.network.trusted_peers.is_empty() {`

is redundant here because the resolve fn will just return Ok(vec![]) if empty

Reply:
got it!